### ISSUE TITLE
Allow users to put text & media in same message

### ISSUE THREAD

**Issue by SD10** [MessageKit/MessageKit#188]
### Mixing MessageData types

MessageKit is limited to sending `.text(String)`, `.attributedText(NSAttributedString)`, and `.photo(UIImage)` messages all as separate messages. Some users may want to send an image with some text.

![b01bd8e4-8f88-11e4-8f00-b2a7e8f51402](https://user-images.githubusercontent.com/7445580/30994925-93f361bc-a47c-11e7-9264-d32421531808.jpg)

Related issues:
[Display Text *and* Media #681](https://github.com/jessesquires/JSQMessagesViewController/issues/681)

**Comment by nathantannar4** [comment]
I'm not sure how I feel about supporting this, as creating an API that can support customizing this merged cell can get quite complex. Things such as does the image have padding?

As this does not effect the core design or add any new features I think it best to focus on occur issues for 2.1

**Comment by SD10** [comment]
Yeah @nathantannar4, I'd agree with removing this from any milestone in the near future.

I know this is something that my company would benefit from having in MessageKit. However, it seems pretty complex for us to support this kind of UI given the current state of things. If someone needs something this complex, they're probably better off just not using MessageKit.

Nonetheless, we can leave it open for visibility/discussion

**Comment by Terens777** [comment]
@SD10  Greetings, is there any news on this issue? And yet, does it make sense not to use MessageKIT, or maybe there are some ways how to solve this situation in your framework?
